src/wazuh_db/wdb_syscollector.c

```diff
@@ -1043,3 +1063,186 @@ int wdb_process_delete(wdb_t * wdb, const char * scan_id) {
 
     return 0;
 }
+
+int wdb_syscollector_processes_save2(wdb_t * wdb, const cJSON * attributes)
+{
+    const char * scan_id = "0";
+    const char * scan_time = cJSON_GetStringValue(cJSON_GetObjectItem(attributes, "scan_time"));
```

Comment:
This needs better sanitization, if the json is incomplete or an arbitrary json is intentionally sent, it would cause a segfault.